### PR TITLE
Remove inactive user from github workflow [skip ci]

### PR DESCRIPTION
**Pull request by sameerz**
remove pxli from blossom-ci.yml

### CODE REVIEW

**Review by revans2 — DISMISSED** [review]

File: .github/workflows/blossom-ci.yml
Comment (jlowe) [inline .github/workflows/blossom-ci.yml:1]: 2024 copyrights

**Review by gerashegalov — APPROVED** [review]

**Comment by gerashegalov** [comment]
build

**Comment by NvTimLiu** [comment]
Should target to `branch24.02`, and auto-merge will update the change to `branch-24.04`

File: .github/workflows/blossom-ci.yml [hunk 1]
```diff
@@ -56,7 +56,6 @@ jobs:
       wbo4958,\
       wjxiz1992,\
       sperlingxx,\
-      pxLi,\
```
Comment (NvTimLiu) [inline .github/workflows/blossom-ci.yml:59]: Should target to branch24.02, and auto-merge will update the change to branch-24.04

**Review by jlowe — APPROVED** [review]
lgtm other that the question of whether this should target 24.02.

File: .github/workflows/blossom-ci.yml (hunk 1 above)
Comment (sameerz) [reply .github/workflows/blossom-ci.yml:59]: This can apply to 24.04 onwards.

**Pull request merged**